packages/carousel/src/react/control.tsx

```diff
@@ -43,45 +43,36 @@ interface ControlProps extends Omit<HTMLPropsFor<'button'>, 'onClick'> {
   direction: ValueOf<typeof vars.controlDirections>
   onClick?: (evt?: React.MouseEvent) => void
 }
+
 interface ControlStatics {
   directions: typeof vars.controlDirections
 }
+
 type ControlComponent = React.FC<ControlProps> & ControlStatics
 export const Control: ControlComponent = props => {
   const { direction, onClick, ...rest } = props
   const context = React.useContext(CarouselContext)
   const themeName = useTheme()
 
-  const {
-    activePage = 0,
-    pageCount = 0,
-    next,
-    prev,
-    setTransitioning
-  } = context
-
   const isPrev = direction === Control.directions.prev
-  const visible = isPrev ? activePage > 0 : activePage !== pageCount - 1
+  const isVisible = isPrev ? context.isPrevVisible : context.isNextVisible
+  const label = isPrev ? 'Previous items' : 'Next items'
```

Comment:
Is this better language than what we previously had for accessibility?

Reply:
I don't know that it's better or not. It matches the Facebook naming. At the least, "items" seems better than "page # of #" since the concept of pages has been changed and diminished.